src/lib/litegraph/src/widgets/ComboWidget.ts

```diff
@@ -131,7 +143,31 @@ export class ComboWidget
     const values = this.getValues(node)
     const values_list = toArray(values)
 
-    // Handle center click - show dropdown menu
+    // Use addItem to solve duplicate filename issues
+    if (this.options.getOptionLabel) {
+      const menuOptions = {
+        scale: Math.max(1, canvas.ds.scale),
+        event: e,
+        className: 'dark',
+        callback: (value: string) => {
+          this.setValue(value, { e, node, canvas })
+        }
+      }
+      const menu = new LiteGraph.ContextMenu([], menuOptions)
+
+      for (const value of values_list) {
+        try {
+          const label = this.options.getOptionLabel(String(value))
+          menu.addItem(label, value, menuOptions)
```

Comment:
Note: one of the downsides of this approach is we don't get the fuzzy search filter because that requires setting the array directly.

Talked it over with @AustinMroz and we do think this is the easiest way to map duplicate labels to the correct value